src/services/dashboards/widgets/__tests__/widget-helper.test.ts

```diff
@@ -34,4 +37,11 @@ describe('[Widget Helper] getWidgetConfig', () => {
         const mergedCostPieConfig = getWidgetConfig(costPieConfigId);
         expect(mergedCostPieConfig.widget_options_schema?.schema?.properties['filters.provider']).toBeTruthy();
     });
+    it('Merge default_properties of base trend config', () => {
+        expect(baseTrendWidgetConfig?.widget_options_schema?.default_properties).toMatchObject(['group_by']);
+        const mergedBaseTrendConfig = getWidgetConfig(costTrendConfigId);
+        expect(mergedBaseTrendConfig?.widget_options_schema?.default_properties).toEqual(expect.arrayContaining([
+            'group_by', 'filters.provider', 'filters.project_id', 'filters.service_account_id',
+        ]));
+    });
```

Comment:
Are we currently creating only tests for widget components?

Reply:
This is test for hook! 
Originally, some hooks had tests.